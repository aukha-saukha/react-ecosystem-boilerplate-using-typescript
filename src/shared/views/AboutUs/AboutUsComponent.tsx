import * as React from 'react';

const AboutUsComponent: React.FC<{}> = (): JSX.Element => (
  <main className="font-size-16" data-test="main">
    About us page
  </main>
);

export default AboutUsComponent;
