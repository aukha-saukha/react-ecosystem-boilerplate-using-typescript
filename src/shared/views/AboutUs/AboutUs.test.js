import { shallow } from 'enzyme';
import React from 'react';

import AboutUs from './AboutUsComponent';

describe('"About Us" component', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<AboutUs />);

    expect(wrapper).toMatchSnapshot();
  });
});
