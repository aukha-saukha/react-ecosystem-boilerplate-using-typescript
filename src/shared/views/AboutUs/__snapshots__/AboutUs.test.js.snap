// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`"About Us" component Renders correctly 1`] = `
<main
  className="font-size-16"
  data-test="main"
>
  About us page
</main>
`;
